registration/include/pcl/registration/sample_consensus_prerejective.h

```diff
@@ -285,8 +299,21 @@ namespace pcl
         * \param inliers indices of source point cloud inliers
         * \param fitness_score output fitness score as RMSE 
         */
+      PCL_DEPRECATED(1, 15, "Thread-unsafe `getFitness` is deprecated")
```

Comment:
@PointCloudLibrary/maintainers Should it be deprecated or should we add a note in the docstring about this "quirk"?

If it should be deprecated, please modify the message to specify the migration path. An example would be:
```cpp
PCL_DEPRECATED(1, 15, "Please use `getFitness(final_transformation_, inliers, fitness_score)` instead")
```

Reply:
I don't have much opinion, but I prefer deprecation in this case because it is an unused protected method, and we may want to delete it if an ABI breakage is allowed.
Anyways, I updated the deprecation message as in the example.